CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
   Bolt's help text has been reformatted to be more consistent with the formatting in other Puppet tools.
 
+* **SSH transport sets `sudo-password` to the same value as `password` by default** ([#1425](https://github.com/puppetlabs/bolt/issues/1425))
+
+  If `sudo-password` is not set when using `run-as`, Bolt will set the value of `sudo-password` to match the value of `password`. It is gated on the future config option, and will be available by default in Bolt 2.0.
```

Comment:
It's not clear what the "it" is in the beginning of the second sentence. Does "It is gated on the future configuration option" mean `sudo-password` will be gated on the future configuration option? If possible can we identify the "it"? Thanks!

Reply:
Updated to say "this behavior" instead.

Reply:
Perfect.